packages/core/src/indexing/postgres/__tests__/postgres-index-api.test.ts

```diff
@@ -51,6 +53,16 @@ describe('init', () => {
       expect(created).toEqual(tableNames)
     })
 
+    test('table creation is idempotent', async () => {
+      const modelsToIndex = [Model.MODEL, StreamID.fromString(STREAM_ID_A)]
+      const indexApi = new PostgresIndexApi(dbConnection, modelsToIndex, true)
+      await indexApi.init()
+      await indexApi.init()
```

Comment:
duplication of init.

Reply:
that's intentional, it's checking that you can init twice without an error.  The double-init is the main part of the test

Reply:
added a comment